core.go

```diff
@@ -594,6 +594,15 @@ func DFT(src Mat, dst Mat) {
 	C.Mat_DFT(src.p, dst.p)
 }
 
+// Calculates a mean and standard deviation of array elements.
```

Comment:
In Go, comments start with the identifier, e.g., "MeanStdDev calculates a mean..."